pom.xml

```diff
@@ -369,6 +369,16 @@
 				<artifactId>hapi-structures-v26</artifactId>
 				<version>2.1</version>
 			</dependency>
+			<dependency>
+				<groupId>dom4j</groupId>
+				<artifactId>dom4j</artifactId>
+				<version>1.6.1</version>
+			</dependency>
+			<dependency>
+				<groupId>jaxen</groupId>
+				<artifactId>jaxen</artifactId>
+				<version>1.2.0</version>
+			</dependency>
 			<dependency>
```

Comment:
just commented on two things that cannot be automatically migrated to JUnit 5 with Intellijs migration tools

Reply:
thanks, Ivo